packages/os/kdump-tmpfiles.conf

```
@@ -0,0 +1,4 @@
+d /local/boot 0700 root root -
+d /local/var/log/kdump 0700 root root -
```

I'd prefer to use the "normal" names for these paths:
```suggestion
d /var/log/kdump 0700 root root -
```